app/assets/stylesheets/static.css

```diff
@@ -0,0 +1,9 @@
+
+
+/*body {
+	display: flex;
+}*/
+
+nav .navbar{
+	background-color: blue;
+}
```

Comment:
So whats the difference between static.css and custom.scss because you have the same styles in them?